src/pages/user/login/service.ts

```diff
@@ -0,0 +1,13 @@
+import request from '@/utils/request';
+import { FormDataType } from './index';
+
+export async function fakeAccountLogin(params: FormDataType) {
+  return request('/api/login/account', {
+    method: 'POST',
+    data: params,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'data' is not sorted alphabetically](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)